generatePlayCards.py

```diff
@@ -34,21 +34,23 @@ def generate_qr_code(url, file_path, icon_path, icon_image_cache={}):
             img = qr.make_image(image_factory=StyledPilImage, embeded_image_path=icon_path)
     img.save(file_path)
 
-def add_qr_code_with_border(c, url, position, box_size, icon_path):
+def add_qr_code_with_border(c, url, position, box_size, noborder, icon_path):
     hash_object = hashlib.sha256(url.encode())
     hex_dig = hash_object.hexdigest()
 
     qr_code_path = f"qr_{hex_dig}.png"  # Unique path for each QR code
     generate_qr_code(url, qr_code_path, icon_path)
     x, y = position
     c.drawImage(qr_code_path, x, y, width=box_size, height=box_size)
-    # c.rect(x, y, box_size, box_size)
+    # if noborder == False :
```

Comment:
I do not think it makes ever sense to print a border here, that is why I disabled it before already